src/models/users.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose'
+
+const userSchema = mongoose.Schema({
+    firstName: { type: String },
+    lastName: { type: String },
+    email: { type: String },
+    password: { type: String },
+    // A user can create many list of todos
+    listTodos: [
```

Comment:
Go through your models and device a more efficient way to create the relationships between them. Hint: You dont need to reference child and parent.